### DISCUSSION TITLE
Can't fetch custom social media links by API

### DISCUSSION THREAD

**Discussion by kecrily** [community/community#46737] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

On February 2 the GitHub blog announced support for custom social media links. However, neither the GraphQL nor the REST APIs can currently fetch a user's custom social media links.

```grahpql
query { 
  user(login: "kecrily") {
   # ...
  }
}
```

<https://api.github.com/users/kecrily>

Please provide support for this.

**Comment by derekprior** [comment]
We haven't updated the API yet, but we do have a backlog item for it.

**Comment by kecrily** [comment, marked as answer]
If anyone is still concerned about this, please note that it was released back in March.

Ref https://github.com/orgs/community/discussions/5720#discussioncomment-5314598
